src/diffusers/image_processor.py

```diff
@@ -364,3 +364,272 @@ def postprocess(
             return self.numpy_to_pil(image), self.numpy_to_depth(image)
         else:
             raise Exception(f"This type {output_type} is not supported")
+
+
+class BlipImageProcessor(ConfigMixin):
+    """
+    Image processor for VAE.
```

Comment:
Needs to change.

Reply:
Could we not reuse https://huggingface.co/docs/transformers/main/en/model_doc/blip#transformers.BlipImageProcessor ? Let's outline the differences from what's provided in `transformers` and see how we can reuse it as much as possible.

Reply:
We can maybe create a new processor, which has a `BlipImageProcessor` object, and use it and add center_crop on top of it

Reply:
> We can maybe create a new processor, which has a BlipImageProcessor object, and use it and add center_crop on top of it

To me this sounds very good. 

Ccing @yiyixuxu for the best way forward here. 

Maybe also cc @amyeroberts to check if she has any other inputs to reuse things from `transformers` here.

Reply:
There was some recent refactoring of the image processors, and now all image processors have `center_crop` as a method (which uses the functionality in the `image_transforms` module). So you should be able to use / copy the BlipImageProcessor directly :)